sdk/storage/Azure.Storage.Files.DataLake/README.md

```diff
@@ -233,7 +233,7 @@ additional questions or comments.
 <!-- LINKS -->
 [source]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Files.DataLake/src
 [package]: https://www.nuget.org/packages/Azure.Storage.Files.DataLake/
-[docs]: https://azure.github.io/azure-sdk-for-net/storage.html
+[docs]: https://docs.microsoft.com/en-us/dotnet/api/azure.storage.files.datalake?view=azure-dotnet-preview
```

Comment:
Can we drop the `view=` query params on these so we don't have to update them in the future?  I think it'll default to latest if not provided.

Reply:
Updated